packages/kbn-rrule/rrule.ts

```diff
@@ -16,6 +16,7 @@ export enum Frequency {
   DAILY = 3,
   HOURLY = 4,
   MINUTELY = 5,
+  SECONDLY = 6,
```

Comment:
Looks like we use `Freq` from 0 to 6 in everywhere but this lib doesn't support the 6th (secondly) interval option, therefore i added it here.